core_backend/app/contents/routers.py

```diff
@@ -32,6 +33,12 @@ async def create_content(
     Create content endpoint. Calls embedding model to get content embedding and
     inserts it to PG database
     """
+    is_tag_valid, content_tags = await validate_tags(
+        user_db.user_id, content.content_tags, asession
+    )
+    if not is_tag_valid:
+        raise HTTPException(status_code=400, detail=f"Invalid tags id: {content_tags}")
```

Comment:
/nit does this work for many tags at once? then we should say "Invalid tag ids: " instead of tags id. Same with the next one